spec/chromium-spec.js

```diff
@@ -571,15 +571,13 @@ describe('chromium feature', () => {
   })
 
   describe('window.opener', () => {
-    let url = `file://${fixtures}/pages/window-opener.html`
```

Comment:
The `url` was used in the both test in this `describe`.
So now in the second test it resolves to `undefined`.
Can you please fix it? It should be the last fix to get green tests.

Reply:
restored